### PR TITLE
#91: Add new exploit example

### LINKED ISSUE #91: Modify CEL-3
## General Comment

CEL-3

**Type of Comment**:
T

**Proposed Change**:
Add Ruxcon research to exploit examples for this threat:
http://www.theregister.co.uk/2016/10/23/every_lte_call_text_can_be_intercepted_blacked_out_hacker_finds/

**Justification**:
Adds further context to the relevance of this threat.

### PR DESCRIPTION
**Pull request by cjb9**
Resolved #91

### CODE REVIEW

**Comment by cjb9** [comment]
@mrdrewkeller Help! Not sure what's causing the build to fail.

File: _cellular-threats/CEL-3.md [hunk 1]
```diff
@@ -7,9 +7,14 @@
         - "3G Security: Security Threats and Requirements (Release 4) [^165]"
         - "LTE Architecture Overview and Security Analysis (Draft NISTIR 8017) [^166]"
         - "LTE Security and Protocol Exploits [^167]"
-    ExploitExample: "Researchers exploit cellular tech flaws to intercept phone calls [^168]"
+    ExploitExample:
+        - "Researchers exploit cellular tech flaws to intercept phone calls [^168]"
+        - "Every LTE call, text, can be intercepted, blacked out, hacker finds" [^244]
```
Comment (mrdrewkeller) [inline _cellular-threats/CEL-3.md]: The reference must be inside of the quotation marks here.

**Comment by mrdrewkeller** [comment]
Looks like there are a few minor formatting issues in CEL-3.

On line 12, the reference must be put inside of the quotations.

In the PossibleCountermeasures section, the key value pairs have to be formatted a little differently now. I've pasted a modified version below.

``` YAML
    ExploitExample:
        - "Researchers exploit cellular tech flaws to intercept phone calls [^168]"
        - "Every LTE call, text, can be intercepted, blacked out, hacker finds [^244]"
    CVEExample:
    PossibleCountermeasures:
        "Ensure baseband firmware prevents the use of insecure cellular encryption algorithms":
            - Original Equipment Manufacturer
        "Use of application layer encryption technologies":
            - Mobile Network Operator
```

**Comment by cjb9** [comment]
@mrdrewkeller thanks. just merged.

**Pull request merged**